journal/modernized/02-journal.adoc

```diff
@@ -4533,9 +4576,14 @@ were sworn against us.`'`"
 Thus I have abstracted the substance and greatest part of our original answer,
 which was signed by Robert Duncon, George Whitehead, and Robert Grassingham.
 
+--
```

Comment:
I think we need to move this closing `--` up a few lines to _before_ `Thus I have abstracted...`